media-libs/libheif/metadata.xml

```diff
@@ -7,13 +7,13 @@
 	<flag name="dav1d">Enable support for the dav1d codec decoding using <pkg>media-libs/dav1d</pkg></flag>
 	<flag name="de265">Enable H.265 support using <pkg>media-libs/libde265</pkg></flag>
 	<flag name="gdk-pixbuf">Enable <pkg>x11-libs/gdk-pixbuf</pkg> plugin</flag>
-	<flag name="go">Enable building <pkg>dev-lang/go</pkg> code/examples</flag>
+	<flag name="gui">Build heif-view (requires USE=tools)</flag>
```

Comment:
I'd probably go for `REQUIRED_USE` here, since otherwise it is quite confusing. Or at least big fat `ewarn`.